sdk/rust-examples/fd_create/test.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# This test script is not ready to be released!
```

Comment:
Why is this file needed?

Reply:
It's not really needed any more as I've included the program in the standard tests.

Reply:
I could delete it?